react-graphql-apollo-tutorial.md

```diff
@@ -1206,21 +1204,21 @@ const Profile = ({ data, loading, error }) => {
+ Apollo Client doesn't update the count of stars after the mutation, though. Normally, it is assumed that the count of stars increments by one when it is starred, with the opposite for unstarring. Since we don't return a count of stargazers in the mutation result, you have to handle the update in Apollo Client's cache yourself.  Using Apollo Client's `refetchQueries` option is the naive approach for a mutation call, or a Mutation component to trigger a refetch for all queries, where the query result might be affected by the mutation. But that's not the best way to deal with this problem. It costs  another query request to keep the data consistent after a mutation. In a growing application, this approach will eventually become problematic. Fortunately, the Apollo Client offers other functionalities to read/write manually from/to the cache locally without more network requests. The Mutation component offers a prop where you can insert update functionality that has access to the Apollo Client instance for the update mechanism.
 
-Before implementing the update functionality for the local state management, let's refactor another piece of code which is later on useful for the desired local state update mechanism. The query definition next to your Profile component has already grown to a large number of fields with multiple object nestings. Previously you have learned about GraphQL fragments and how they can be used to split out parts of a query in order to reuse them later on. Let's do it by splitting out all the field information that you have used for a repository's node. You can define this fragment in a *src/Repository/fragments.js* file to keep it reusable for other components.
+Before implementing the update functionality for the local state management, let's refactor another piece of code that will be useful for a local state update mechanism. The query definition next to your Profile component has grown to several fields with multiple object nestings. Previously, you learned about GraphQL fragments, nd how they can be used to split parts of a query to reuse later.  Next, we will split all the field information you used for the repository's node. You can define this fragment in the *src/Repository/fragments.js* file to keep it reusable for other components.
```

Comment:
"about GraphQL fragments, nd how they can" to "about GraphQL fragments, and how they can"

Reply:
"that will be useful for a local state update mechanism" to "that will be useful for such a local state update mechanism"

Reply:
"such" is unnecessary.  It's all part of the same sentence, so the user knows you're talking about update functionality for the local state management.  "Such" would only be proper if you started a new sentence.

Reply:
Okay 👍